config/cmake/addBoost.cmake

```diff
@@ -61,170 +70,111 @@ if(WIN32 AND NOT UNIX_LIKE)
             endif()
         endforeach()
     endforeach()
-
+    message(STATUS "${boost_paths}")
     find_path(
         BOOST_TEST_PATH
         NAMES boost/version.hpp
         HINTS ENV BOOST_INSTALL_PATH
         PATHS ${BOOST_INSTALL_PATH} ${boost_paths}
     )
-
     if(BOOST_TEST_PATH)
-        set(Boost_ROOT ${BOOST_TEST_PATH})
-    endif(BOOST_TEST_PATH)
-else()
-    if(NOT Boost_ROOT)
-        if(BOOST_INSTALL_PATH)
-            set(Boost_ROOT "${BOOST_INSTALL_PATH}")
-        elseif($ENV{BOOST_INSTALL_PATH})
-            set(Boost_ROOT "$ENV{BOOST_INSTALL_PATH}")
+        if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.30)
+            find_path(
+                BOOST_CMAKE_PATH
+                NAMES BoostConfig.cmake
+                PATHS ${BOOST_TEST_PATH}/${BOOST_MSVC_LIB_PATH}/cmake
+                PATH_SUFFIXES
+                    Boost-1.86.0
+                    Boost-1.85.0
+                    Boost-1.84.0
+                    Boost-1.83.0
+                    Boost-1.82.0
+                    Boost-1.81.0
+                    Boost-1.80.0
+                    Boost-1.79.0
+                    Boost-1.78.0
+                    Boost-1.77.0
+                    Boost-1.76.0
+                    Boost-1.75.0
```

Comment:
Should this list also include 1.74 and 1.73?

Reply:
The BoostConfig didn't work very well until 1.75.   So maybe we could add those if CMAKE <3.30.

Reply:
Never mind I checked 1.74 and it looks like it would most likely work.   I will add them.